__tests__/app.test.js

```diff
@@ -2,9 +2,63 @@ const pool = require('../lib/utils/pool');
 const setup = require('../data/setup');
 const request = require('supertest');
 const app = require('../lib/app');
+const Vacations = require('../lib/models/Vacations');
+const VacationService = require('../lib/services/VacationService');
 
 describe('ct-lab09 routes', () => {
   beforeEach(() => {
     return setup(pool);
   });
+
+  it('post will create a new vacation in the vacations table', async () => {
+    const res = await request(app)
+      .post('/api/v1/vacations')
+      .send({ destination: 'Hawaii', startDate: 'May 5th', endDate: 'May 20th', details: 'going for anniversary celebration' });
+    expect(res.body).toEqual({ id: expect.any(String), destination: 'Hawaii', startDate: 'May 5th', endDate: 'May 20th', details: 'going for anniversary celebration', photo: expect.any(String) });
+  });
+
+  it('get vacation by id 1 will return correct vacation', async () => {
+    await VacationService.new({ destination: 'Hawaii', startDate: 'May 5th', endDate: 'May 20th', details: 'going for anniversary celebration' });
```

Comment:
This works! You can also add a `beforeEach` that inserts a new vacation, that way you won't need to insert a new vacation inside each `it()` :)